test/unit/merge.js

```diff
@@ -270,9 +270,11 @@ describe('Merge', function () {
         should(sideEffects).deepEqual({
           savedDocs: [{
             _id: initialFile._id,
+            class: 'application',
```

Comment:
Do we really want to hardcode mime/class default values everywhere?

Why not use `initialFile.class` here?